app/(golems)/golems/components/SkillsShowcase.tsx

```
@@ -391,7 +392,10 @@ function FeaturedSkill() {
       </div>
 
       {/* Terminal content */}
-      <div className="scrollbar-none h-[320px] overflow-y-auto p-4 font-mono text-xs leading-relaxed text-[#c0b8a8] md:h-[380px] md:px-5 md:text-[0.76rem]">
+      <div
+        className="scrollbar-none h-[320px] overflow-y-auto p-4 font-mono text-xs leading-relaxed text-[#c0b8a8] md:h-[380px] md:px-5 md:text-[0.76rem]"
+        aria-label="Terminal demonstration: skill installation process"
+      >
```

Biome (2.4.6)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)